### DISCUSSION TITLE
Option in settings to auto unload pinned/essential tabs

### DISCUSSION THREAD

**Discussion by JJ21112** [zen-browser/desktop#4025] [Ideas]
This has been an issue for quite a few people in the community because we use multiple workspaces and have pinned tabs as resources - which then causes lots of tabs to build up and be permanently open, causing performance issues.
Idea 1: An option to set an unload time for essential/pinned tabs
Idea 2: An option to unload all of the tabs from the workspace you switch from

**Comment by mitr0001** [comment]
Idea 1: I understand that not unloading pinned/essential tabs is a kind of feature, but I would love it to be implemented like in this suggestion. I would like it because for me, after visiting my sites that are pins and essentials, manual unloading would not be necessary and could happen within the specified time. It is reminiscent of Arc and how you can click the - button to unload a tab (would love that as a feature)

**Comment by syscat29** [comment]
There is an option to manually unload tabs, for now you can use [SuperPins](https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc)

**Reply by mitr0001** [reply]
Yes, I use SuperPins and manually unload with the keyboard shortcut. I just think Idea 1 above is a good one and would really improve the interface (in my opinion)

**Reply by mitr0001** [reply]
*Having the timer for pinned/essential tabs could really improve the experience and having a (-) button could really improve teh interface

**Comment by timrekelj** [comment]
I +1 the idea 2 because the browser takes to much resources if I forget to unload memory heavy tabs like Figma before I switch to another workspace

**Reply by timrekelj** [reply]
Also, can this be developed as Zen mod? I understand that developers are overwhelmed so I can try creating the mod if it is possible

**Comment by bwnbrg** [comment]
This feels like a critical feature for me. My computer has slowed to a snails pace as I've started using Zen and I'm hoping I don't have to switch to another browser. Was using Sidekick before they shut down and their tab offload worked pretty well.

**Reply by JJ21112** [reply]
Idk if I'm allowed to plug my extension but I made it to combat this issue essentially so feel free to use it or not until something similar is officially added

https://addons.mozilla.org/en-GB/firefox/addon/zen-tab-unloader/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=collection

**Reply by bwnbrg** [reply]
@JJpro1234 Awesome - thanks for sharing. Does it fix the issue of non-pinned/essential tabs remaining loaded forever too? #8822

**Comment by cherrynoize** [comment]
Definitely feel like I need a lot more of the features which Sidebery or Auto Tab Discard offer.

**Comment by lnnrthnng** [comment]
Any news on this?

**Reply by JJ21112** [reply]
Unfortunately not

**Comment by turbanisch** [comment]
The only missing feature that keeps me from switching from Arc. I'd suggest adding a setting for a specific reset time in addition to an interval.
